bundles/af-core/src/test/resources/schema/0.12.5/adaptive-form-aem-responsive-grid-properties.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$id": "https://git.corp.adobe.com/pages/livecycle/af2-docs/schema/generated/adaptive-form-aem-responsive-grid-properties.schema.json",
```

Comment:
This is incorrect, please check the earlier version of schema on how the id is created, don't reference internal links in github